R/btw_client.R

```diff
@@ -136,7 +136,7 @@ btw_client <- function(..., client = NULL, tools = NULL, path_btw = NULL) {
 
   if (!skip_tools) {
     maybe_quiet <- if (is.null(tools)) suppressMessages else I
-    maybe_quiet(btw_register_tools(client, tools = config$tools))
+    maybe_quiet(client$set_tools(tools = btw_tools(config$tools)))
```

Comment:
Oh shoot, we need to do a bit more work here. There was a lot of discussion at the time, but Hadley settled on `$set_tools()` doing exactly that: setting _all of the tools_. So in order to keep existing tools on the `client`, we'll need to `$get_tools()`, update that list, and then call `$set_tools()` with that merged list.

(Technically, because tools are [registered in a for loop](https://github.com/tidyverse/ellmer/blob/fed4fc372a0d3b369da8d1d883fafc407655c7fe/R/chat.R#L450-L454), you could just append btw's tools to the tool list.)